katello/katello-backup

```diff
@@ -160,7 +160,7 @@ else
   run_cmd("tar --selinux --create --gzip --file=config_files.tar.gz --listed-incremental=.config.snar #{CONFIGS.join(' ')} 2>/dev/null", [0,2])
   puts "Done."
 
-  `shopt -s dotglob ; cp #{@options[:incremental]}/*.snar .` if @options[:incremental]
+  `shopt -s dotglob ; cp #{@options[:incremental]}/.*.snar .` if @options[:incremental]
```

Comment:
This way the wildcard won't get expanded and the copy will fail with `cp: cannot stat ‘.../.*.snar’ : No such file or directory`
I suggest using `FileUtils.cp`, fileutils are already improted in the script